### PR TITLE
Refactor JPA 2.1 Spec CDI FAT

### PR DESCRIPTION
**Pull request by jgrassel**
Refactor the bucket, separating the test grouping into EJB and Web groups, to differentiate the names enough so that there is no confusion recognizing names waiting for apps to start.

### CODE REVIEW

**Comment by jgrassel** [comment]
#build

File: dev/com.ibm.ws.jpa_spec21_fat/fat/src/com/ibm/ws/jpa/TestCDI_WEB.java [hunk 1]
```diff
@@ -37,12 +37,13 @@
...
-public class TestCDI extends JPAFATServletClient {
+public class TestCDI_WEB extends JPAFATServletClient {
     private final static String CONTEXT_ROOT = "TestCDISimple";
+    private final static String CONTEXT_ROOT_JPALIB = "TestCDIWithJPALib";
     private final static String RESOURCE_ROOT = "test-applications/CDI/";
     private final static String appFolder = "apps";
-    private final static String appName = "TestCDISimple";
-    private final static String appNameEar = appName + ".ear";
+//    private final static String appName = "TestCDISimple";
+//    private final static String appNameEar = appName + ".ear";
```
Comment (dazey3) [inline dev/com.ibm.ws.jpa_spec21_fat/fat/src/com/ibm/ws/jpa/TestCDI_WEB.java]: Maybe remove these commented out names?

File: dev/com.ibm.ws.jpa_spec21_fat/test-applications/CDI/apps/TestCDIWithJPALibEJB.ear/lib/jpamodel.jar/META-INF/persistence.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- 
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
```
Comment (dazey3) [inline dev/com.ibm.ws.jpa_spec21_fat/test-applications/CDI/apps/TestCDIWithJPALibEJB.ear/lib/jpamodel.jar/META-INF/persistence.xml]: 2020

File: dev/com.ibm.ws.jpa_spec21_fat/fat/src/com/ibm/ws/jpa/TestCDI_WEB.java (hunk 1 above)
Comment (jgrassel) [reply dev/com.ibm.ws.jpa_spec21_fat/fat/src/com/ibm/ws/jpa/TestCDI_WEB.java]: Addressed

File: dev/com.ibm.ws.jpa_spec21_fat/test-applications/CDI/apps/TestCDIWithJPALibEJB.ear/lib/jpamodel.jar/META-INF/persistence.xml (hunk 2 above)
Comment (jgrassel) [reply dev/com.ibm.ws.jpa_spec21_fat/test-applications/CDI/apps/TestCDIWithJPALibEJB.ear/lib/jpamodel.jar/META-INF/persistence.xml]: Addressed

**Comment by jgrassel** [comment]
#run-libby-bot

**Pull request merged**